### ISSUE TITLE
Identify plugin UI enhancement by moving "Help" button to top toolbar and merging QHBoxLayouts of "View" and "Mode" 

### ISSUE THREAD

**Issue by imanhn** [qgis/QGIS#39842]
I believe "Identify plugin" is one of the most useful plugins and so every tiny change to it would be reasonable.

In order to have more space in QTreeWidget's (feature list), I would like to suggest to merge "View" and "Mode" QHBoxLayouts into one QHBoxLayout (See the the following Picture).

I think it is also better to move that ugly "Help" button (bottom right) to the top toolbar.

If others think it is a good call, I would send my pull request (already did in my repo).

@m-kuhn @PeterPetrik 

![UI_enhancement_identify](https://user-images.githubusercontent.com/9729773/98374394-d69f3880-2055-11eb-9591-29d60a1098e5.png)

**Comment by m-kuhn** [comment]
Looks like a good suggestion to me, a pull request will be welcome !

**Comment by imanhn** [comment]
Thanks.
I will send it ASAP.

**Comment by rduivenvoorde** [comment]
Interesting, eager to see this. 
The actual results is still a tree? 
I do miss the 'settings' toolbutton ( in which I just added something recently :-) ) in your picture (mockup?)

**Comment by imanhn** [comment]
@rduivenvoorde 
Richard,
Thanks,
Yes I saw your changes, and they are still there. (the right picture is taken from Qt designer).

I did not change the tree part. It is what it was. I have just merged the QHBoxLayout at the bottom of the form into one and moved "Help" to the top.

No new feature has been added and the the Tree is still there with those three options applicable to raster layers.

I have a new feature suggestion which I have implemented. Nothing big but our customers wanted that so I thought it could be handy to other customers as well. I will open up a new issue for that so we can discuss it there.
